packages/medusa-plugin-twilio-sms/src/services/twilio-sms.js

```diff
@@ -0,0 +1,31 @@
+import { BaseService } from "medusa-interfaces"
+import twilio from "twilio"
+
+class TwilioSmsService extends BaseService {
+  /**
+   * @param {Object} options - options defined in `medusa-config.js`
+   *    e.g.
+   *    {
+   *      account_sid: "1234",
+   *      auth_token: "XXX",
+   *      from_number: "+4512345678"
+   *    }
+   */
+  constructor({}, options) {
+    super()
+
+    this.options_ = options
+
+    this.twilioClient = twilio(options.account_sid, options.auth_token)
+  }
+
+  async sendSms(to, body) {
+    return this.twilioClient.messages.create({
+      to,
+      body,
+      from: this.options_.from_number,
+    })
+  }
+}
```

Comment:
Maybe make this as generic as possible. I.e.:
```js
  async sendSms(config) {
    const toSend = {
        from: this.options_.from_number,
        ...config
    }
    return this.twilioClient.messages.create(toSend)
  }
```
This way we default to the `from_number` in `options` but we give the flexibility to do other things with the plugin if needed